packages/bitcore-wallet-service/src/lib/fiatrateservice.ts

```diff
@@ -154,6 +153,47 @@ export class FiatRateService {
     );
   }
 
+  getRates(opts, cb) {
+    $.shouldBeFunction(cb);
+
+    opts = opts || {};
+    const rates = [];
+
+    const now = Date.now();
+    const coin = opts.coin;
+    const ts = _.isNumber(opts.ts) ? opts.ts : now;
```

Comment:
no need to support both array and number, that was made for compatibility on old clients.